tests/resources/test_record_resource.py

```diff
@@ -168,6 +168,23 @@ def test_publish_draft(client, input_data):
     _assert_single_item_response(response)
 
 
+def test_search_draft(client, input_data):
+    response = client.post(
+        "/mocks", data=json.dumps(input_data), headers=HEADERS)
+    recid = response.json['id']
+
+    assert response.status_code == 201
+    _assert_single_item_response(response)
+
+    response = client.get("/mocks?q=", headers=HEADERS)
```

Comment:
This test is failing because we are hitting the published record endpoints and there are no published records; there is only a draft at this point. Publishing and searching is a test case already covered in invenio-records-resources, so I wouldn't worry about adding this test here. The `q=` case is (or should be) in invenio-records-resources I think.